tests/cases/fourslash/completionInFunctionLikeBody.ts

```diff
@@ -19,7 +19,7 @@ verify.completions(
         includes: ["async", "await"].map(
             name => ({ name, sortText: completion.SortText.GlobalsOrKeywords })
         ),
-        excludes: ["public", "private", "protected", "constructor", "readonly", "static", "abstract", "get", "set"],
+        excludes: ["public", "private", "protected", "constructor", "static", "abstract", "get", "set"],
```

Comment:
Why is `readonly` ok in at position 1 and 2 ?

Reply:
we allow now type keywords there and readonly is a type keyword with things like `readonly T[]`